cmd/kyma/alpha/create/module/moduleconfig.go

```diff
@@ -0,0 +1,149 @@
+package module
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/blang/semver/v4"
+	"gopkg.in/yaml.v3"
+)
+
+type Config struct {
```

Comment:
I added some comments to the descriptions in `module.go`. Should I copy those here as well?

Reply:
no, this file is not documentation source, there are only code-level docs (comments) here